sysutils/bats-core/Portfile

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem      1.0
+PortGroup       github 1.0
+
+github.setup    bats-core bats-core 1.4.1 v
+categories      sysutils
+platforms       darwin
+supported_archs noarch
+license         MIT
+maintainers     nomaintainer
+description     Bash Automated Testing System
+long_description \
+    Bats is a TAP-compliant testing framework for Bash. It provides a simple way to \
+    verify that the UNIX programs you write behave as expected.
+
+github.tarball_from archive
+
+checksums       rmd160  86429672c062f4736d49adaaf7302e31ab853f59 \
+                sha256  bff517da043ae24440ec8272039f396c2a7907076ac67693c0f18d4a17c08f7d \
+                size    98964
+
+patchfiles      patch-test-bats.diff \
+                patch-test-parallel.diff
+
+depends_run     port:coreutils
```

Comment:
Perhaps `bash` will suffice as a dependency?

Reply:
Appears true that coreutils no longer required and is sensible to depend bash.